.shippable.yml

```diff
@@ -65,11 +66,18 @@ build:
             ./scripts/ci/check-compliance.py --commits ${COMMIT_RANGE} || true;
           fi;
       - >
+          python2 ./scripts/ci/get_modified_tests.py --commits origin/${PULL_REQUEST_BASE_BRANCH}..HEAD;
```

Comment:
suggest having the modified_test.args file being printed on the stdout of get_modified_tests to have clarity on where it comes from:
python2 ./scripts/ci/get_modified_tests.py ... > modified_tests.args
if [ -s modified_tests.args ]; then
   blah blah
fi